### PR TITLE
Update instructions for OCaml 5.1.0 and Eio.0.11.

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

File: meio.opam [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 opam-version: "2.0"
 synopsis: "Monitor live Eio programs"
 description:
   "Eio-console provides an executable that allows you to monitor OCaml programs using Eventring."
 maintainer: ["patrick@sirref.org"]
 authors: ["Patrick Ferris"]
 license: "MIT"
-homepage: "https://github.com/patricoferris/eio-console"
-bug-reports: "https://github.com/patricoferris/eio-console/issues"
+homepage: "https://github.com/tarides/meio"
```
Comment (patricoferris) [inline meio.opam:8]: The repo moved since this PR :)) 

```suggestion
homepage: "https://github.com/ocaml-multicore/meio"
```

**Review by patricoferris — APPROVED** [review]
Thanks @tmcgilchrist !

**Comment by tmcgilchrist** [comment]
Applied the opam file changes, should be good to go.
 
**Note** this uses a rebased version of [TheLortex/eio#runtime-events](https://github.com/TheLortex/eio/tree/runtime-events) vs [TheLortex/eio#runtime-events-tracing](https://github.com/TheLortex/eio/tree/runtime-events-tracing) as used in https://github.com/ocaml-multicore/eio/pull/554. Meio needs an update to target whatever version ends up in Eio.

**Comment by tmcgilchrist** [comment]
Merging. This PR brings support up to Eio.0.11 and OCaml 5.1 which is still behind the latest Eio but easier to test out than previously.

**Pull request merged**